config/settings.yml

```diff
@@ -15,6 +15,10 @@ feature_flags:
     local: true
     staging: true
     production: true
+  evidence_upload:
```

Comment:
It's ok leaving the feature configuration but don't think we need to worry about using it right now. Only way to hit the page should be manually knowing the URL.